docs/CHANGELOG.md

```diff
@@ -15,6 +15,14 @@ Changes are grouped as follows:
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+
+## [0.60.0] - 26-01-24
+
+## Added
+
+- Introduced `Rules` profiling which follow data modeling lifecycle
```

Comment:
We are not officially introducing this feature yet. 
```suggestion
```

Reply:
done !